Makefile

```diff
@@ -34,20 +34,24 @@ RUN = $(DOCKER) run \
   --rm \
   -i $(ENABLE_TTY) `$(MAKE_DOCKER)`
 
+BUILD_TYPE ?= Debug
+DOCKER ?= y
```

Comment:
Instead of using `y` or `n`, I would like to use `1` or `0` to be consistent with the other makefile options.

Reply:
Done.